Jenkinsfile_CNP

```diff
@@ -40,7 +40,6 @@ withPipeline(type, product, component) {
   env.PACT_BROKER_URL = "pact-broker.platform.hmcts.net"
   env.PACT_BROKER_PORT = "443"
   env.PACT_BROKER_SCHEME = "https"
-  env.WA_TASK_MANAGEMENT_API_URL = "http://wa-task-management-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```

Comment:
This wasn't resolving anyway, just that java 17 didn't care much, while java 21 properly checks and was failing on this url.